enhancements/sig-architecture/34-clusterset-delete/README.md

```diff
@@ -0,0 +1,73 @@
+# ManagedClusterSet Delete policy
+ 
+## Release Signoff Checklist
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management/website/)
+ 
+## Summary
+The proposed work enhances the managedClusterSet API to handle managedClusterSet Delete policy.
+ 
+## Motivation
+Currently, when we add a cluster to a clusterset, we will add a label `cluster.open-cluster-management.io/clusterset:<Clusterset Name>` to the cluster. But when the clusterset is deleted, this label will not be removed.
+ 
+So we need to design the delete policy for clusterset.
+ 
+## Proposal Detail
+Actually, we always use clusterset as a group of clusters. So when deleting a clusterset, there are some options for the delete policy.
+1. When clusterset deleted, delete the clusterset labels in managedClusters.
```

Comment:
I still think this is an unnatural behavior to remove a thing not managed by it.